sandboxes/analysis/output_error_monitor.lua

```diff
@@ -0,0 +1,57 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Monitors Parquet output plugin for errors, alerting when they pass a given threshold.
+
+```lua
+filename = 'parquet_output_error_monitor.lua'
+message_matcher = 'Type == "hindsight.plugins"'
+ticker_interval = 60
+preserve_data = false
+process_message_inject_limit = 1
+
+```
+--]]
+local alert      = require "heka.alert"
+local alert_percent = alert.thresholds.output.percent
+local minimum_sample = alert.thresholds.output.minimum_sample
+assert(type(minimum_sample) == "number" and minimum_sample >= 0,
+               string.format("alert: 'minimum_sample' must be a non-negative number", k))
+
+assert(type(alert_percent) == "number" and alert_percent > 0 and alert_percent <= 100,
+       string.format("alert: 'percent' must contain a percentage of message failures (1-100)", k))
+
+-- INJECT_MESSAGE_COUNT = 0
```

Comment:
At some point we will want the ability to monitor any combination of these fields driven by the cfg